test/units/testsuite-post.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+set -eux
+set -o pipefail
+
+TEST_NUM=${1?}
+echo "Running post script for TEST-${TEST_NUM}"
+
+(! journalctl -q -o short-monotonic --grep "didn't pass validation" >> /failed)
```

Comment:
What about replacing the inline shell script in the `end.service` with a standalone script and adding this check into it? So we don't have to hook up another script to the test services (which is easily forgotten).

Reply:
(Or add it directly to `check_results_common` and `check_result_{nspawn,qemu}_unittests`)

Reply:
Let's try with end.service.